platform/qt/include/QMapLibreGL/settings.hpp

```diff
@@ -0,0 +1,112 @@
+#ifndef QMAPLIBREGL_SETTINGS_H
+#define QMAPLIBREGL_SETTINGS_H
+
+#include <QImage>
+#include <QString>
+
+#include <functional>
+
+#include "export.hpp"
+
+// This header follows the Qt coding style: https://wiki.qt.io/Qt_Coding_Style
+
+// TODO: this will be wrapped at some point
```

Comment:
I would open an issue instead of the TODO. This makes it easier for new people to understand what open tasks there are and gives a bit more context.

Reply:
But feel free to leave the todo here if it helps your workflow... Maybe just open an issue in addition...

Reply:
Opened a ticket: https://github.com/maplibre/maplibre-gl-native/issues/348

Will leave also TODO in the code.